pyaedt/edb_core/edb_data/simulation_configuration.py

```diff
@@ -14,17 +14,18 @@
 from pyaedt.generic.constants import validate_enum_class_value
 
 try:
+    from System import String
     from System.Collections.Generic import Dictionary
 except ImportError:
     if os.name != "posix":
         warnings.warn("This module requires pythonnet.")
 
 
 class SimulationConfiguration(object):
-    """Parses an ASCII simulation configuration file, which supports all types of inputs
+    """ASCII simulation configuration file parser, which supports all types of inputs
     for setting up and automating any kind of SI or PI simulation with HFSS 3D Layout
     or Siwave. If fields are omitted, default values are applied. This class can be instantiated directly from
```

Comment:
```suggestion
    """Provides an ASCII simulation configuration file parser.
    
    This parser supports all types of inputs for setting up and automating any kind
    of SI or PI simulation with HFSS 3D Layout or Siwave. If fields are omitted, default
    values are applied. This class can be instantiated directly from
```

Reply:
Alex says that the descriptions of classes and methods have to be kept short so they can fully display in tables and summary pages. I've reformatted the content to do this.

Reply:
Thanks. Good point